waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoredAspectTest.java

```diff
@@ -109,6 +115,23 @@ public void monitorSuccesses() throws Throwable {
     assertThat(rs.timer().count(), is(1L));
   }
 
+  @Test
+  public void monitorSuccessWithTags() throws Throwable {
```

Comment:
We're not testing the tags, please add asserts that the correct tags are stored.